lua/metals.lua

```diff
@@ -465,4 +466,12 @@ M.toggle_setting = function(setting)
   end
 end
 
+M.select_test_suite = function()
+  test_explorer.dap_select_test_suite(conf.get_config_cache().root_dir)
```

Comment:
I mentioned this in the other file, but I don't think we actually need to include the `root_dir` here. Instead we should rely on the `targetUri` that comes with the `metals-update-test-explorer` command.